src/main/java/com/spotinst/sdkjava/model/ElastigroupConverter.java

```diff
@@ -1216,6 +1212,175 @@ private static ApiScalingAction toDal(ScalingAction scalingAction) {
         return retVal;
     }
 
+    public static ApiElastigroupStartDeployment toDal(ElastigroupStartDeployment elastigroupDeployment) {
+        ApiElastigroupStartDeployment retVal = null;
+
+        if (elastigroupDeployment != null) {
+            retVal = new ApiElastigroupStartDeployment();
+
+            if (elastigroupDeployment.isBatchSizePercentageSet()) {
+                retVal.setBatchSizePercentage(elastigroupDeployment.getBatchSizePercentage());
+            }
+
+            if (elastigroupDeployment.isDrainingTimeoutSet()) {
+                retVal.setDrainingTimeout(elastigroupDeployment.getDrainingTimeout());
+            }
+
+            if (elastigroupDeployment.isGracePeriodSet()) {
+                retVal.setGracePeriod(elastigroupDeployment.getGracePeriod());
+            }
+
+            if (elastigroupDeployment.isHealthCheckTypeSet()) {
+                retVal.setHealthCheckType(elastigroupDeployment.getHealthCheckType().getName());
+            }
+
+            if (elastigroupDeployment.isStrategySet()) {
+                retVal.setStrategy(toDal(elastigroupDeployment.getStrategy()));
+            }
+
+        }
+
+        return retVal;
+    }
+
+    private static ApiElastigroupStrategy toDal(ElastigroupStartDeploymentStrategy elastigroupStrategy) {
+        ApiElastigroupStrategy retVal = null;
+
+        if (elastigroupStrategy != null) {
+            retVal = new ApiElastigroupStrategy();
+
+            if (elastigroupStrategy.isActionSet()) {
+                retVal.setAction(elastigroupStrategy.getAction().getName());
+            }
+
+            if (elastigroupStrategy.isBatchMinHealthyPercentageSet()) {
+                retVal.setBatchMinHealthyPercentage(elastigroupStrategy.getBatchMinHealthyPercentage());
+            }
+
+            if (elastigroupStrategy.isOnFailureSet()) {
+                retVal.setOnFailure(toDal(elastigroupStrategy.getOnFailure()));
+            }
+        }
+
+        return retVal;
+    }
+
+    private static ApiElastigroupStartDeploymentOnFailure toDal(ElastigroupStartDeploymentOnFailure elastigroupOnFailure) {
```

Comment:
rename input parameter to strategyOnFailure

Reply:
Done